src/pages/workspaces/workspace/Data.js

```diff
@@ -846,9 +846,11 @@ const WorkspaceData = _.flow(
             }, [
               [
```

Comment:
Or... is this array necessary?

Reply:
wow, good catch - I hadn't noticed that nested array. Updated the PR.

Reply:
Nice. React key warnings mean a component is being passed an array as a child. There are plenty of valid use cases for that (lists created by mapping over an array of objects, etc.), but if that seems odd, then look out for unnecessary array wrappers.